### DISCUSSION TITLE
How can I skip motion correction?

### DISCUSSION THREAD

**Discussion by beanei** [flatironinstitute/CaImAn#1359] [Q&A]
Dear Everyone,

Supper beginner question here: I am recording 2p calcium imaging and due to my setup, I practically have no movement or drift in my videos. Therefore I figured it would make sense to skip the motion correction part of CaImAn. However, when I try to move to the next step (CNMF), it requires items defined in the motion correction segment, such as "correlation_image". Is there an efficient way to turn off the need for motion correction?

Thanks all

**Comment by pgunn** [comment]
@beanei Ah, sorry I didn't notice this until now; yes, it is possible to skip motion correction; I believe demo_pipeline_cnmfE shows how to do it. Give it a look and let me know if that helps.
